01week/dataTypes.js

```diff
@@ -74,3 +74,28 @@ parseInt(string);
 // this does not work because no matter what you write in the prompt the computer reads it as a string
 var object = prompt("Write an examply of a datatype");
 document.write(typeof object);
```

Comment:
we're running this in node, so we wouldn't need to touch the doc. To test, you could just console log.